cypresse2e/cypress/integration/e2eTests/privacyPolicy/privacyPolicy.spec.js

```diff
@@ -0,0 +1,142 @@
+        validateEachPara(privacyPolicyContent.paraPolicyCx[12]);
+      });
+
+      it("Get and Validate the PRIVACY POLICY content for the TENTH paragraph with Order List starting with-10. With Whom We Share Your Personal Data?", (ParaNum = 110) => {
+        validateEachPara(privacyPolicyContent.paraPolicyCx[13]);
+      });
+      it("Get and Validate the PRIVACY POLICY content for the ELEVENTH paragraph with Order List starting with-11. How long We store your Personal Data?", (ParaNum = 111) => {
+        validateEachPara(privacyPolicyContent.paraPolicyCx[14]);
+      });
+
+      it("Get and Validate the PRIVACY POLICY content for the TWELVETH paragraph with Order List starting with-12. Where We Store Your Personal Data", (ParaNum = 112) => {
+        validateEachPara(privacyPolicyContent.paraPolicyCx[15]);
+      });
+      it("Get and Validate the PRIVACY POLICY content for the THIRTEENTH paragraph with Order List starting with-13. How We Protect Your Personal Data", (ParaNum = 113) => {
+        validateEachPara(privacyPolicyContent.paraPolicyCx[16]);
+      });
+
+      it("Get and Validate the PRIVACY POLICY content for the FOURTEENTH paragraph with Order List starting with-14. Contact Us", (ParaNum = 114) => {
+        validateEachPara(privacyPolicyContent.paraPolicyCx[17]);
+      });
+      it("Get and Validate the PRIVACY POLICY content for the FIFTEENTH paragraph with Order List starting with-15. Changes to Our Privacy Policy", (ParaNum = 115) => {
+        validateEachPara(privacyPolicyContent.paraPolicyCx[18]);
+      });
+      it("Get and Validate Date of Document created", () => {
+        privacyPolicy
+          .getDateDocumented()
+          .invoke("text")
+          .should("eq", dateOfDocument);
+      });
+    },
+  );
+  context(
+    "Check on FightPandemics Logo and Email links on the Privacy Policy Page for FightPandemics",
+    () => {
+      beforeEach(() => {
+        privacyPolicy.visit();
+      });
+
+      it("FP logo is visible and clickable", () => {
+        cy.checkFpLogoIsVisibleAndClickable(logo.getFpLogoLocator());
+      });
+
+      it("Emails are visible with correct link", () => {
+        var numEmails = privacyPolicyContent.paraPolicyWithEmail.length;
+
+        for (var i = 0; i < numEmails; i++) {
+          checkEmaiEachParagraph(
+            privacyPolicyContent.paraPolicyWithEmail[i].id,
+            privacyPolicy.legalEmail,
+            privacyPolicy.legalEmailRef,
+          );
+        }
+      });
+    },
+  );
+
+  //to validate each paragraph existing
+  function validateEachPara(objData) {
+    const id = objData.id;
+    const text = objData.text;
+    privacyPolicy.getLocationPara(id).contains(text);
+  }
+
+  //to check email each paragraph
+  function checkEmaiEachParagraph(paraNum, email, link) {
```

Comment:
here small spelling: checkEmai**l**..

Reply:
@galeza done :).  I also update improve the function name and the use of parameter. TY